processes/development/new-app-setup/README.md

```diff
@@ -275,17 +265,13 @@ Atlas is the best option if wishing to scale CPU & Memory independently to it's
     1. NOTE: Paul needs to do this step currently. Will fix...
 1. Edit Build Settings:
     1. Go to [https://semaphoreci.com/okgrow/](https://semaphoreci.com/okgrow/)&lt;PROJECT_NAME&gt;/settings
-    1. Set the Node version to node.js 4.8.0
-    1. Add each of these lines under setup
-        1. `curl https://install.meteor.com/ | sh`
-        1. `meteor --version`
-        1. `meteor npm install`
-        1. `meteor npm start` _wip: find and add instructions to listen for success and exit process_
+    1. Set the Node version to node.js 7.10.1 (or later)
```

Comment:
As of Meteor 1, can we change this to 8.7.x now?

Reply:
@karldanninger Yes.